test/shared/helpers/forEach.test.js

```diff
@@ -103,11 +103,11 @@ describe('forEach', function () {
       var thisCall = spy.getCall(0),
           args = thisCall.args[0];
 
-      expect(args).to.deep.property('key', 0);
```

Comment:
:+1: seems legit - didn't think about that one.  updated it and the rest of the cases that i *can* there are some things that i'm test for deep equal that can't be compared this way :(